### DISCUSSION TITLE
UpdateHub v2.2.3

### DISCUSSION THREAD

**Discussion by onyx-nxt** [NexovaDev/UpdateHub#266] [Announcements]
## Version
`2.2.3`

# Changelog
- Should've finally fixed the timeout error once and for all
- Fixed `SyntaxError: "undefined" is not valid JSON` error (fixes #248)
- Fixed `TypeError: Cannot read properties of undefined (reading 'replace')` error (fixes #247)
- Fixed auto update of UpdateHub not working when updating via popup warning (fixes #254)
- Using strict matching to prevent potential errors
- And more bug fixes

# How to update
Click the Update Available button in the sidebar. (if you don't see it, force close the app by pressing the shift key, clicking the X icon in the windows title bar, and then relaunch the app.)
<img src="https://user-images.githubusercontent.com/71038229/229697691-43ef66a0-a29d-4e06-af4b-4df0ab5eb772.png" width="572" />
Then wait for it to download the update file, and it will install automatically. You now have the latest version!

### VirusTotal scan
This app is not a virus, trust me. If you want, here's a [VirusTotal scan](https://www.virustotal.com/gui/file/80a081b63fac71e96930c24c342ed51a184e9bca964633d7d898b07b82f90b92/ "VirusTotal Scan"). As you can see, no anti-virus software has flagged it as a virus. The executable below's SHA-256 Hash is `80A081B63FAC71E96930C24C342ED51A184E9BCA964633D7D898B07B82F90B92`, which matches the VirusTotal scan.

### Remember, if you encounter any bugs in UpdateHub, report them so we can fix them!

# Thanks for downloading!

### SHA256
`80A081B63FAC71E96930C24C342ED51A184E9BCA964633D7D898B07B82F90B92`

<hr /><em>This discussion was created from the release <a href='https://github.com/KK-Designs/UpdateHub/releases/tag/v2.2.3'>UpdateHub v2.2.3</a>.</em>

**Comment by WarningU2** [comment]
Thank you.  With the demise of SUMO this looks like a fine replacement.  I'll trial it for a week or so and then contribute.   Thanks again
